packages/documentation/plugins/github-api-pages/gatsby-node.js

```diff
@@ -1,39 +1,38 @@
-const path  = require('path');
+const path = require('path');
```

Comment:
linting issues in this file got into the repo because the precommit hook was not failing properly.